packages/blocks/list-operations/src/lib/actions/extract-from-action.ts

```diff
@@ -1,12 +1,13 @@
 import { BlockAuth, createAction, Property } from '@openops/blocks-framework';
+import { getItemsAsArray } from '../utils';
 
 export const extractFromListAction = createAction({
   auth: BlockAuth.None(),
   name: 'extract_from_list_action',
   description: 'Extract items from a list with a given key',
   displayName: 'Extract From List',
   props: {
-    listItems: Property.Array({
+    listItems: Property.ShortText({
```

Comment:
shouldnt it be long text?

Reply:
Hm i think we might aswell support it, i was just thinking bc if its a text field i'd expect to be able to also manually use it

Reply:
fair point I'll change